flow/connectors/clickhouse/qrep.go

```diff
@@ -78,116 +46,3 @@ func (c *ClickHouseConnector) getTableSchema(ctx context.Context, tableName stri
 
 	return rows.ColumnTypes(), nil
 }
-
-func (c *ClickHouseConnector) IsQRepPartitionSynced(ctx context.Context,
-	req *protos.IsQRepPartitionSyncedInput,
-) (bool, error) {
-	queryString := fmt.Sprintf(`SELECT COUNT(*) FROM %s WHERE partitionID = '%s'`, qRepMetadataTableName, req.PartitionId)
-
-	var count uint64
-	if err := c.database.QueryRow(ctx, queryString).Scan(&count); err != nil {
-		return false, fmt.Errorf("failed to execute query: %w", err)
-	}
-	return count > 0, nil
-}
-
-func (c *ClickHouseConnector) SetupQRepMetadataTables(ctx context.Context, config *protos.QRepConfig) error {
-	err := c.createQRepMetadataTable(ctx)
-	if err != nil {
-		return err
-	}
-
-	if config.WriteMode.WriteType == protos.QRepWriteType_QREP_WRITE_MODE_OVERWRITE {
-		err = c.execWithLogging(ctx, "TRUNCATE TABLE "+config.DestinationTableIdentifier)
-		if err != nil {
-			return fmt.Errorf("failed to TRUNCATE table before query replication: %w", err)
-		}
-	}
-
-	return nil
-}
-
-func (c *ClickHouseConnector) createQRepMetadataTable(ctx context.Context) error {
-	// Define the schema
-	schemaStatement := `
-	CREATE TABLE IF NOT EXISTS %s (
-		flowJobName String,
-		partitionID String,
-		syncPartition String,
-		syncStartTime DateTime64,
-		syncFinishTime DateTime64
-		) ENGINE = MergeTree()
-		ORDER BY partitionID;
-	`
-	queryString := fmt.Sprintf(schemaStatement, qRepMetadataTableName)
-	err := c.execWithLogging(ctx, queryString)
-	if err != nil {
-		c.logger.Error("failed to create table "+qRepMetadataTableName,
-			slog.Any("error", err))
-
-		return fmt.Errorf("failed to create table %s: %w", qRepMetadataTableName, err)
-	}
-	c.logger.Info("Created table " + qRepMetadataTableName)
-	return nil
-}
-
-func (c *ClickHouseConnector) ConsolidateQRepPartitions(_ context.Context, config *protos.QRepConfig) error {
-	c.logger.Info("Consolidating partitions noop")
-	return nil
-}
-
-// CleanupQRepFlow function for clickhouse connector
-func (c *ClickHouseConnector) CleanupQRepFlow(ctx context.Context, config *protos.QRepConfig) error {
-	c.logger.Info("Cleaning up flow job")
-	return c.dropStage(ctx, config.StagingPath, config.FlowJobName)
-}
-
-// dropStage drops the stage for the given job.
-func (c *ClickHouseConnector) dropStage(ctx context.Context, stagingPath string, job string) error {
```

Comment:
does external store handle this?

Reply:
added back, bit weird since we skip consolidate, but stages are tied to consolidation because of snowflake